### PR TITLE
Add dedicated page to updating px on k8s

### PR DESCRIPTION
**Pull request by harsh-px**
#Screenshots of the generated pages:

px-k8s-update-1

k8s-update-2

### CODE REVIEW

File: scheduler/kubernetes/update.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* Alternately, you can also change the image in the DaemonSet spec file and apply it using `$ kubectl apply -f <px-spec.yaml>`.
+
+### 3. Update Portworx pods
+
+It is not recommended to delete the Portworx pod while an application is actively issuing I/O. This can induce race conditions in docker causing it to hang. 
+
+The following procedure should be followed:
+1. Cordon the node where you want to upgrade Portworx: `$ kubectl cordon <node-name>`
+2. Delete application pods running on this node that are using Portworx volumes: `$ kubectl delete pod <pod-name>`
+    * Since application pods are expected to be managed by a controller like `Deployement` or `StatefulSet`, Kubernetes will spin up a new replacement pod on another node.
```
Comment (msh100) [inline scheduler/kubernetes/update.md]: s/Deployement/Deployment/

**Comment by msh100** [comment]
@harsh-px is there any intention of getting this merged in? Seems like a valuable document to have

**Comment by ferrantim** [comment]
@harsh-px can I close this?

**Comment by harsh-px** [comment]
Upgrade page was merged outside of the PR . Hence closing.

**Pull request closed without merging**